src/common/lambda.rs

```diff
@@ -143,66 +140,86 @@ impl fmt::Display for Lambda {
                 Opcode::Con => {
                     let (constant_index, consumed) = build_number(&self.code[index..]);
                     index += consumed;
-                    writeln!(f, "Load Con\t{}\t{:?}", constant_index, self.constants[constant_index])?;
```

Comment:
Breaking this across multiple lines does not improve readability. This whole expression probably needs some cleanup, though.